File: Company-Wise-Questions/Knoldus/palindrome_creator.py
def palindromeCreator(str):
    n = len(str)
    if(str == str[::-1] and len(str)>2):
        return "palindrome"
    
    for i in range(0,n):
        oneL = str[0:i] + str[i+1:]
        if (oneL == oneL[::-1] and len(oneL)>2):
            return str[i]
    
    for i in range(0,n):
        oneL = str[0:i] + str[i+1:]
        for j in range(i,n):
            twoL = oneL[0:j] + oneL[j+1:]
            if (twoL == twoL[::-1] and len(twoL)>2):
                return str[i] + oneL[j]
    
    return "not possible"

File: Company-Wise-Questions/Knoldus/palindrome_creator.js
function PalindromeCreator(givenString) {
    if ((givenString === givenString.split('').reverse().join('')) && givenString.length>2){
        return 'Palindrome' 
    }
    
    for (let x = 0; x < givenString.length; x++) {
        const stringMinusOne = givenString.slice(0, x) + givenString.slice(x + 1);
        if ((stringMinusOne === stringMinusOne.split('').reverse().join('')) && stringMinusOne.length>2) {
          return givenString[x];
        }
    }

    for (let x = 0; x < givenString.length; x++) {
        const stringMinusOne = givenString.slice(0, x) + givenString.slice(x + 1);
        for (let y = x; y < stringMinusOne.length; y++) {
            const stringMinusTwo = stringMinusOne.slice(0, y) + stringMinusOne.slice(y + 1);
            if((stringMinusTwo === stringMinusTwo.split('').reverse().join('')) && stringMinusTwo.length>2){ 
                return givenString[x] + stringMinusOne[y];
            }
        }
    }
    return 'not possible';
}